<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostPyqWidget</class>
 <widget class="QWidget" name="PostPyqWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PostPyqWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QTextEdit" name="m_contentTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>390</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #000000;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>内容</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QWidget" name="m_imageWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>390</width>
     <height>130</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_publishPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>390</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8fc0e7;</string>
   </property>
   <property name="text">
    <string>发布</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
